apps/files/src/default.js

```diff
@@ -31,11 +32,18 @@ const navItems = [{
 
 const routes = [{
   path: '',
-  redirect: `/${appInfo.id}/list/home`
+  redirect: `/${appInfo.id}/list/home`,
+  components: {
```

Comment:
why are here two components - I don't get this ....

Reply:
it's like a global vue-router managed slot.

Reply:
so the route controls the views and their content .... processing ..... :robot: 

THX